clients/privacy-center/cypress/support/commands.ts

```diff
@@ -51,12 +51,6 @@ declare global {
        * implement using regular browser interactions, as this provides less end-to-end coverage.
        *
        * https://www.cypress.io/blog/2018/11/14/testing-redux-store/#dispatch-actions
-       *
-       * TODO(ssangervasi): Make the action object less permissive. Right now it can be `AnyAction`
```

Comment:
Unrelated, but I removed this TODO- I think the types here are reasonable and I prefer a zero-TODO policy on main when possible 😄